specification/cosmos-db/data-plane/Microsoft.Tables/preview/2019-02-02/examples/TableQuery.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Any particular reason we added this example when one for `TableGet` already exists with the same contents?

Reply:
It was misnamed - the previous one was actually TableQuery